### DISCUSSION TITLE
V3 Pro-Account Feature "alle Icon Sets (ohne Pro nur Material Design Icons)"

### DISCUSSION THREAD

**Discussion by Frank-2021** [Zefau/ioBroker.jarvis#1521] [Q&A]
Hallo,

wo und wie kann ich eigene Icons einbinden? Wo müssen die auf dem IOBroker System liegen?

Danke 

Frank

**Comment by mcuiobroker** [comment]
https://mcuiobroker.gitbook.io/jarvis-infos/jarvis/beispiele/module/geraete-einstellungen/icons-eigene/eigene-icons
@Zefau 
```
alle Icon Sets (ohne Pro nur Material Design Icons) -> von iconify ergänzen?
```

**Comment by Zefau** [comment]
Aktuell nur als base64. Es gibt ein Feature Request auch Grafiken einzubinden. Dies wird aber erst mit v3.1 kommen.

**Comment by Frank-2021** [comment]
Hallo,

 

ich habe mal anbei ein paar Bilder und ein Video vom Desktop (sind beim ersten Versuch nicht bei Dir angekommen – daher hier: https://wolke.eissnet.de/s/NmXRNRejxjYGdPe

 

Wenn ich einen Schalten betätige, springt der wieder von selber zurück und es wird nichts geschalten.

Bei den Tastern Garage passiert auch nichts.

Und die Sensor-Eingänge reagieren auch nicht.

 

Im „alten“ Jarvis hat alles funktioniert – ebenso in VIS.

 

Grüße

 

Frank

 

 

Von: Zefau ***@***.***> 
Gesendet: Donnerstag, 20. Januar 2022 14:01
An: Zefau/ioBroker.jarvis ***@***.***>
Cc: Frank-2021 ***@***.***>; Author ***@***.***>
Betreff: Re: [Zefau/ioBroker.jarvis] V3 Pro-Account Feature "alle Icon Sets (ohne Pro nur Material Design Icons)" (Discussion #1521)

 

Aktuell nur als base64. Es gibt ein Feature Request auch Grafiken einzubinden. Dies wird aber erst mit v3.1 kommen.

—
Reply to this email directly, view it on GitHub <https://github.com/Zefau/ioBroker.jarvis/discussions/1521#discussioncomment-2006224> , or unsubscribe <https://github.com/notifications/unsubscribe-auth/ATEHZJO7IEFWU73ORY3FSRTUXABSLANCNFSM5MG7Q55A> .
Triage notifications on the go with GitHub Mobile for iOS <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>  or Android <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub> .
